src/containers/pages/NotFound.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+const NotFound = () => (
```

Comment:
Chimba de componente

Reply:
Jajajajaajja, que pereza hacer una pagina para decir 404: Not Found, pero si, ahí iría una pagina que se vea decente